src/components/map/index.js

```diff
@@ -7,7 +7,8 @@ import Component from './component';
 const mapStateToProps = state => ({
   ...state.map,
   mapStyle: mapStyle(state),
-  mapboxApiAccessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN
+  mapboxApiAccessToken: process.env.REACT_APP_MAPBOX_ACCESS_TOKEN,
+  isCollapse: state.layers.isCollapse
```

Comment:
the one that exists in layers store is `isCollapsed`, the first time is set to flase but trigger some error...